src/main/java/com/rusticisoftware/tincan/Attachment.java

```diff
@@ -79,8 +86,23 @@ public Attachment(JsonNode jsonNode) throws URISyntaxException, MalformedURLExce
         if (! fileUrlNode.isMissingNode()) {
             this.setFileUrl(new URL(fileUrlNode.textValue()));
         }
+
+        JsonNode contentNode = jsonNode.path("content");
+        if (! contentNode.isMissingNode()) {
+            this.setContent(contentNode.binaryValue());
+        }
```

Comment:
I'm not convinced this should be coming from the passed in JsonNode, since the content is never part of the JSON itself. It feels like if we want to pass attachment content at construction time it makes more sense to pass it as an additional argument and add the new constructor signature for that case. I could be convinced I'm wrong if you'd like to try.